data/scenarios/02Tutorials/07-install.yaml

```diff
@@ -0,0 +1,71 @@
+name: Install
+description: |
+  Learn how to install devices and gain new capabilities.
+entities:
+  - name: Goal
+    display:
+      attr: device
+      char: 'X'
+    description:
+    - You know how to 'grab' things lying around, so you are ready to get
+      an upgrade!
+    - You learn new capabilites which allow you to perform more complex commands
+      by installing devices.
+    - There is a 3D printer lying nearby, go 'grab' it.
```

Comment:
Is there something else we could use instead of 3D printer here?  Maybe we could have them try to do `make` to see that they need a certain device installed in order to be able to do it, then `grab` and `install` a workbench.  It seems better to connect it back to something they have already done rather than using a new, unknown device to illustrate a new concept.

Reply:
Oh, I fully intend it to lead directly into the next challenge.
- in this last challenge with threads installed on base you obtain the 3D printer and install it
- in all the rest you use the 3D printer to build robots

How about this:
> There is a highly useful device called "3D printer" lying nearby, go 'grab' it.

Reply:
Yeah, that sounds good.